syncstorage/src/tokenserver/handlers.rs

```diff
@@ -220,3 +220,13 @@ pub async fn heartbeat(db: Box<dyn Db>) -> Result<HttpResponse, Error> {
         }
     }
 }
+
+/// Generates an error to test the Sentry integration
+pub async fn test_error() -> Result<HttpResponse, TokenserverError> {
```

Comment:
From what I could tell, [these two lines](https://github.com/mozilla-services/syncstorage-rs/blob/master/syncstorage/src/web/handlers.rs#L704-L705) weren't necessary to generate a test error, but I could definitely be wrong here